.eslintrc.json

```diff
@@ -19,7 +19,7 @@
     "@typescript-eslint/no-floating-promises": "off",
     "no-redeclare": "off",
     "no-unused-vars": "off",
-    "@typescript-eslint/no-unused-vars": ["error"],
+    "@typescript-eslint/no-unused-vars": ["warn"],
```

Comment:
Is this because of `import type { fastifyCookie } from '@fastify/cookie'` ? Add one `eslint-disable-next-line` instead of changing whole config.